frontend/src/components/Card.jsx

```diff
@@ -1,9 +1,10 @@
 import propTypes from "prop-types";
+import Slugify from "../services/Slugify";
```

Comment:
components and classes should start with an uppercase letter. Slugify is not a component, neither a class : the file name should start with a lowercase letter